mixer/adapter/solarwinds/config/config.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package adapter.solarwinds.config;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "config";
+option (gogoproto.goproto_getters_all) = false;
+option (gogoproto.equal_all) = false;
+option (gogoproto.gostring_all) = false;
+
+message Params {
+    
+    string appoptics_access_token = 1;
+    
+    string papertrail_url = 2;
+
+    string papertrail_local_retention = 3;
```

Comment:
can we mark this as google.protobuf.Duration ?

Reply:
good one. Thx. Will do.

Reply:
if the retention is a duration, can we call this `papertrail_local_retention_duration` ?

Reply:
Sure will do.